FSJ_django20_project/FSJ/templates/FSJ/coord_awards_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends "FSJ/base_with_banner.html" %}
```

Comment:
The indentation of this template is missing. Each nested element should be offset by a consistent amount to allow for readability of a "block" (the contents inside the opening and closing tags)

Reply:
Done